### ISSUE TITLE
0.8 beta 1 -> 0.8 beta 2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,

### ISSUE THREAD

**Issue by rb2k** [blakeblackshear/frigate#414]
**Describe the bug**
After a beta 1 to beta 2 upgrade, I keep seeing these errors in the logs:

```
2020-12-20T16:28:35.800868777Z frigate.video                  INFO    : garage_frigate: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
2020-12-20T16:28:35.800954778Z frigate.video                  INFO    : garage_frigate: ffmpeg process is not running. exiting capture thread...
```

It seems to recover, but this happens pretty often on multiple cameras (different brands/models)
I thought I'd make a quick report since beta 2 is just fresh off the press :)

**Version of frigate**
0.8 beta 2

**Config file**
Include your full config file wrapped in triple back ticks.
```yaml
  garage_frigate:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.30.101/ch0_0.h264
          roles:
            - detect
            - rtmp
    fps: 4
    height: 1080
    width: 1920
    save_clips:
      # Required: enables clips for the camera (default: shown below)
      enabled: True
      # Optional: Number of seconds before the event to include in the clips (default: shown below)
      pre_capture: 10

  frontdoor_frigate:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.30.102/live0
          roles:
            - detect
            - rtmp
    height: 1080
    width: 1920
    fps: 4
    save_clips:
      # Required: enables clips for the camera (default: shown below)
      enabled: True
      # Optional: Number of seconds before the event to include in the clips (default: shown below)
      pre_capture: 10
```

**Logs**
```
ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
```

**Frigate debug stats**
```
{"detection_fps":0.0,"detectors":{"coral":{"detection_start":0.0,"inference_speed":8.72,"pid":42}},"frontdoor_frigate":{"camera_fps":4.1,"capture_pid":58,"detection_fps":0.0,"pid":48,"process_fps":4.1,"skipped_fps":0.0},"garage_frigate":{"camera_fps":4.1,"capture_pid":55,"detection_fps":0.0,"pid":46,"process_fps":4.1,"skipped_fps":0.0},"sqwrlcam2_frigate":{"camera_fps":4.0,"capture_pid":61,"detection_fps":0.0,"pid":51,"process_fps":4.0,"skipped_fps":0.0},"sqwrlcam_frigate":{"camera_fps":4.0,"capture_pid":52,"detection_fps":0.0,"pid":45,"process_fps":4.1,"skipped_fps":0.0}}
```

**FFprobe from your camera**

Run the following command and paste output below

Garage:
```
Input #0, rtsp, from 'rtsp://192.168.30.101/ch0_0.h264':
  Metadata:
    title           : Session streamed by "rRTSPServer"
    comment         : ch0_0.h264
  Duration: N/A, start: 0.049733, bitrate: N/A
    Stream #0:0: Video: h264 (Main), yuvj420p(pc, bt709, progressive), 1920x1080, 20 tbr, 90k tbn, 180k tbc
```

**Computer Hardware**
 - OS: Suse Linux
 - Virtualization: Docker
 - Coral Version: USB
 - Network Setup: Wired (cameras: wifi)

**Camera Info:**
- Yihome indoor + yihome outdoor + eufy 2k

**Additional context**
Worked in beta 1 from what I could tell, error started with beta 2

**Comment by mario-tux** [comment]
The same here with 0.8 beta 2:
```
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
```

**Comment by rb2k** [comment]
Tested with beta 3, same issue.

**Comment by blakeblackshear** [comment]
Update your config to this to see what the ffmpeg error message is:
```yaml
  garage_frigate:
    ffmpeg:
      global_args: -hide_banner -loglevel info
      inputs:
        - path: rtsp://192.168.30.101/ch0_0.h264
          roles:
            - detect
            - rtmp
    fps: 4
    height: 1080
    width: 1920
    save_clips:
      # Required: enables clips for the camera (default: shown below)
      enabled: True
      # Optional: Number of seconds before the event to include in the clips (default: shown below)
      pre_capture: 10

  frontdoor_frigate:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.30.102/live0
          roles:
            - detect
            - rtmp
    height: 1080
    width: 1920
    fps: 4
    save_clips:
      # Required: enables clips for the camera (default: shown below)
      enabled: True
      # Optional: Number of seconds before the event to include in the clips (default: shown below)
      pre_capture: 10
```

**Comment by rb2k** [comment]
Oh, unexpected:

```
2020-12-22T16:26:50.864715273Z ffmpeg.sqwrlcam2_frigate.detect ERROR   : [tcp @ 0x561ed197e340] Connection to tcp://192.168.30.104:554?timeout=5000000 failed: No route to host


2020-12-22T16:26:50.864901251Z ffmpeg.sqwrlcam2_frigate.detect ERROR   : rtsp://192.168.30.104/ch0_0.h264: No route to host


2020-12-22T16:26:50.866010677Z frigate.video                  INFO    : sqwrlcam2_frigate: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures


2020-12-22T16:26:50.866218096Z frigate.video                  INFO    : sqwrlcam2_frigate: ffmpeg process is not running. exiting capture thread...
```

**Comment by blakeblackshear** [comment]
I don't see any changes from beta 1->2 that could have caused issues here.

**Comment by rb2k** [comment]
Ok, beta 3 doesn't show this for 'garage' or the battery swapped other camera.

Not sure what that was :)

**Comment by mario-tux** [comment]
I confirm that I've found (after hours of good work!) Frigate stuck with the following log:
```
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-0: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...,
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures,
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...,
```

I faced them since beta 2 but they could be present even on beta 1. I can't reproduce them in a systematic way.

**Comment by rb2k** [comment]
If you raise the log level, you might see something like this.

For me it's the camera timing out every now and then for a second or two:

```

ffmpeg.garage_frigate.detect   ERROR   : rtsp://192.168.30.101/ch0_0.h264: Connection timed out


frigate.video                  INFO    : garage_frigate: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures


frigate.video                  INFO    : garage_frigate: ffmpeg process is not running. exiting capture thread...


ffmpeg.garage_frigate.detect   ERROR   : [tcp @ 0x55595f1ee7c0] Connection to tcp://192.168.30.101:554?timeout=5000000 failed: Connection timed out


ffmpeg.garage_frigate.detect   ERROR   : rtsp://192.168.30.101/ch0_0.h264: Connection timed out


frigate.video                  INFO    : garage_frigate: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures


frigate.video                  INFO    : garage_frigate: ffmpeg process is not running. exiting capture thread...


ffmpeg.garage_frigate.detect   ERROR   : [tcp @ 0x5586777e07c0] Connection to tcp://192.168.30.101:554?timeout=5000000 failed: Connection timed out


ffmpeg.garage_frigate.detect   ERROR   : rtsp://192.168.30.101/ch0_0.h264: Connection timed out


frigate.video                  INFO    : garage_frigate: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures


frigate.video                  INFO    : garage_frigate: ffmpeg process is not running. exiting capture thread...


ffmpeg.garage_frigate.detect   ERROR   : [tcp @ 0x558007edc7c0] Connection to tcp://192.168.30.101:554?timeout=5000000 failed: Connection timed out


ffmpeg.garage_frigate.detect   ERROR   : rtsp://192.168.30.101/ch0_0.h264: Connection timed out
```

**Comment by mario-tux** [comment]
switching on `debug` logger level on first restart this it the output:
```
    rv = self.handle_user_exception(e),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1821, in handle_user_exception,
    reraise(exc_type, exc_value, tb),
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 39, in reraise,
    raise value,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1950, in full_dispatch_request,
    rv = self.dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1936, in dispatch_request,
    return self.view_functions[rule.endpoint](**req.view_args),
  File "/opt/frigate/frigate/http.py", line 240, in latest_frame,
    frame = current_app.detected_frames_processor.get_current_frame(camera_name, draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 477, in get_current_frame,
    return self.camera_states[camera].get_current_frame(draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 264, in get_current_frame,
    motion_boxes = self.motion_boxes.copy(),
AttributeError: 'CameraState' object has no attribute 'motion_boxes',
frigate.http                   ERROR   : Exception on /camera-1/latest.jpg [GET],
Traceback (most recent call last):,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2447, in wsgi_app,
    response = self.full_dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1952, in full_dispatch_request,
    rv = self.handle_user_exception(e),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1821, in handle_user_exception,
    reraise(exc_type, exc_value, tb),
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 39, in reraise,
    raise value,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1950, in full_dispatch_request,
    rv = self.dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1936, in dispatch_request,
    return self.view_functions[rule.endpoint](**req.view_args),
  File "/opt/frigate/frigate/http.py", line 240, in latest_frame,
    frame = current_app.detected_frames_processor.get_current_frame(camera_name, draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 477, in get_current_frame,
    return self.camera_states[camera].get_current_frame(draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 264, in get_current_frame,
    motion_boxes = self.motion_boxes.copy(),
AttributeError: 'CameraState' object has no attribute 'motion_boxes',
frigate.http                   ERROR   : Exception on /camera-2/latest.jpg [GET],
Traceback (most recent call last):,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2447, in wsgi_app,
    response = self.full_dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1952, in full_dispatch_request,
    rv = self.handle_user_exception(e),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1821, in handle_user_exception,
    reraise(exc_type, exc_value, tb),
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 39, in reraise,
    raise value,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1950, in full_dispatch_request,
    rv = self.dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1936, in dispatch_request,
    return self.view_functions[rule.endpoint](**req.view_args),
  File "/opt/frigate/frigate/http.py", line 240, in latest_frame,
    frame = current_app.detected_frames_processor.get_current_frame(camera_name, draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 477, in get_current_frame,
    return self.camera_states[camera].get_current_frame(draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 264, in get_current_frame,
    motion_boxes = self.motion_boxes.copy(),
AttributeError: 'CameraState' object has no attribute 'motion_boxes',
frigate.http                   ERROR   : Exception on /camera-3/latest.jpg [GET],
Traceback (most recent call last):,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2447, in wsgi_app,
    response = self.full_dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1952, in full_dispatch_request,
    rv = self.handle_user_exception(e),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1821, in handle_user_exception,
    reraise(exc_type, exc_value, tb),
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 39, in reraise,
    raise value,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1950, in full_dispatch_request,
    rv = self.dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1936, in dispatch_request,
    return self.view_functions[rule.endpoint](**req.view_args),
  File "/opt/frigate/frigate/http.py", line 240, in latest_frame,
    frame = current_app.detected_frames_processor.get_current_frame(camera_name, draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 477, in get_current_frame,
    return self.camera_states[camera].get_current_frame(draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 264, in get_current_frame,
    motion_boxes = self.motion_boxes.copy(),
AttributeError: 'CameraState' object has no attribute 'motion_boxes',
frigate.http                   ERROR   : Exception on /camera-5/latest.jpg [GET],
Traceback (most recent call last):,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2447, in wsgi_app,
    response = self.full_dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1952, in full_dispatch_request,
    rv = self.handle_user_exception(e),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1821, in handle_user_exception,
    reraise(exc_type, exc_value, tb),
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 39, in reraise,
    raise value,
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1950, in full_dispatch_request,
    rv = self.dispatch_request(),
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1936, in dispatch_request,
    return self.view_functions[rule.endpoint](**req.view_args),
  File "/opt/frigate/frigate/http.py", line 240, in latest_frame,
    frame = current_app.detected_frames_processor.get_current_frame(camera_name, draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 477, in get_current_frame,
    return self.camera_states[camera].get_current_frame(draw_options),
  File "/opt/frigate/frigate/object_processing.py", line 264, in get_current_frame,
    motion_boxes = self.motion_boxes.copy(),
AttributeError: 'CameraState' object has no attribute 'motion_boxes',
```

**Comment by mario-tux** [comment]
Today Frigate stuck again. With the following logger settings:
```
logger:
  default: debug
  logs:
    peewee: info
```
I was not able to recover any useful log: from docker logs I can get only the tail.
```
frigate.video                  INFO    : camera-0: ffmpeg process is not running. exiting capture thread...
frigate.video                  INFO    : camera-3: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate.video                  INFO    : camera-3: ffmpeg process is not running. exiting capture thread...
frigate.video                  INFO    : camera-5: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate.video                  INFO    : camera-5: ffmpeg process is not running. exiting capture thread...
frigate.video                  INFO    : camera-2: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate.video                  INFO    : camera-2: ffmpeg process is not running. exiting capture thread...
frigate.video                  INFO    : camera-1: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate.video                  INFO    : camera-1: ffmpeg process is not running. exiting capture thread...
frigate.video                  INFO    : camera-4: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate.video                  INFO    : camera-4: ffmpeg process is not running. exiting capture thread...
```

**Comment by ljmerza** [comment]
was this resolved? im getting the same issue on the 0.8 release branch for all my RTSP cameras
